### PR TITLE
Add container

### PR DESCRIPTION
**Pull request by heyanlong**

### CODE REVIEW

**Comment by jmjoy** [comment]
It's better to use [multi-stage builds](https://docs.docker.com/build/building/multi-stage/) in Dockerfile, otherwise the target image is too big.

File: .github/workflows/container.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - name: Docker build
+        run: |
+          cd docker
+          docker build . -t skywalking-php
+          docker tag skywalking-php ghcr.io/apache/skywalking-php:latest
+          docker tag skywalking-php ghcr.io/apache/skywalking-php:v0.1.0
```
Comment (wu-sheng) [inline .github/workflows/container.yml:34]: Does this seem like you are building the image repeatedly?

Comment (wu-sheng) [reply .github/workflows/container.yml:34]: I think you should follow tag or release event of GHA?

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Comment (heyanlong) [reply .github/workflows/container.yml:34]: I want to do it manually, it doesn't seem necessary now

**Comment by wu-sheng** [comment]
If you want to do this manually, you could do it directly locally, I suppose. Setting up a GHA manual task could make this image uploaded repeated, rather than once, this is not expected.

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM php:8.1-fpm-bullseye as builder
+
+LABEL org.opencontainers.image.source=https://github.com/apache/skywalking-php
+LABEL org.opencontainers.image.description="The PHP Agent for Apache SkyWalking, which provides the native tracing abilities for PHP project."
+LABEL org.opencontainers.image.licenses="Apache 2.0"
+
+ARG SKYWALKING_AGENT=0.1.0
```
Comment (jmjoy) [inline docker/Dockerfile:22]: Make this argument configurable rather than dead code?

Comment (heyanlong) [reply docker/Dockerfile:22]: The value `0.1.0` is the default

Comment (jmjoy) [reply docker/Dockerfile:22]: In fact, the default value is not required.

File: docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM php:8.1-fpm-bullseye as builder
+
+LABEL org.opencontainers.image.source=https://github.com/apache/skywalking-php
+LABEL org.opencontainers.image.description="The PHP Agent for Apache SkyWalking, which provides the native tracing abilities for PHP project."
+LABEL org.opencontainers.image.licenses="Apache 2.0"
```
[the remark this answers is not in the document]
Comment (heyanlong) [reply docker/Dockerfile:20]: Yes, should move to second stage

**Review by jmjoy — APPROVED** [review]

**Pull request merged**